core/src/main/java/org/apache/iceberg/encryption/BaseEncryptedOutputFile.java

```diff
@@ -20,12 +20,13 @@
 
 import org.apache.iceberg.io.OutputFile;
 
-class BaseEncryptedOutputFile implements EncryptedOutputFile {
+public class BaseEncryptedOutputFile implements EncryptedOutputFile {
```

Comment:
Why was this made public?

Reply:
to be available in the _envelope_ subpackage; but since we move classes to the encryption package, no need in this change anymore, I'll remove.